src/site/paragraphs/table.drupal.liquid

```diff
@@ -27,31 +27,37 @@
     }
 {% endcomment %}
 
-
-<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table">
-
-    {% if entity.fieldTable.caption %}
-        <caption>{{ entity.fieldTable.caption }}</caption>
-    {% endif %}
-
-    {% for value in entity.fieldTable.value %}
-        {% if forloop.first == true %}
-            <thead>
-                <tr>
-                    {% for column in value %}
-                        <th>{{ column }}</th>
-                    {% endfor %}
-                </tr>
-            </thead>
-        {% elsif forloop.last == false %}
-            <tbody>
-                <tr class="class">
+<table data-template="paragraphs/table" data-entity-id="{{ entity.entityId }}" class="va-table rwd" role="table">
+    <tbody>
+        {% if entity.fieldTable.caption %}
+            <caption>{{ entity.fieldTable.caption }}</caption>
+        {% endif %}
+        {% assign colLabels = entity.fieldTable.value.0  %}
+        {% for value in entity.fieldTable.value %}
+            {% if forloop.first == true %}
+                <thead>
+                    <tr role="row">
+                        {% for column in value %}
+                            <th role="columnheader" scope="col">{{ column }}</th>
+                        {% endfor %}
+                    </tr>
+                </thead>
+            {% elsif forloop.last == false %}
```

Comment:
Won't this make it so that if there is more than length 2 for `entity.fieldTable.value`, the middle values won't be rendered?

Reply:
Wondering if that should be an `else` instead?

Reply:
Oh nevermind seeing now that this is how it was previously, so better not to mess with that lol

Reply:
hehe agreed :)